frontend/src/stimulus/controllers/dynamic/work-packages/activities-tab/index.controller.ts

```diff
@@ -661,10 +664,18 @@ export default class IndexController extends Controller {
         console.error('Error saving activity:', error);
       })
       .finally(() => {
-        this.saveInProgress = false;
+        this.formSubmitInProgress = false;
       });
   }
 
+  private set formSubmitInProgress(inProgress:boolean) {
```

Comment:
:green_circle: I think semantically it would make more sense to me if we used a regular function instead of a setter here. The reason being that it's not obvious that when we do the `formSubmitInProgress = value` that something else is happening. It looks like a flag.

No biggie though, feel free to ignore.

Reply:
Ah, makes sense to me! Updating...